sqe-database-access/EditionRepository.cs

```diff
@@ -393,29 +415,101 @@ public async Task<string> GetDeleteToken(EditionUserInfo editionUser)
                     "an editor with write rights must have read rights"
                 );
             using (var connection = OpenConnection())
+            {
+                // Add the editor
+                var editorInfo = await connection.QuerySingleAsync<DetailedUserWithToken>(
+                    UserDetails.GetQuery(
+                        new List<string> { "user_id", "forename", "surname", "organization" },
+                        new List<string> { "email" }
+                    ),
+                    new { Email = editorEmail }
+                );
+
+                // TODO: Use the database user email token system rather than a JWT for this (see commented code below.
+                // That way, we can query for "open" editor requests. Must wait to update the database before we can
+                // implement this functionality.
+                var secret = _config.GetSection("AppSettings").GetSection("Secret").Value;
+                editorInfo.Token = new JwtBuilder()
```

Comment:
Storing all the permission information in the token is nice, but I guess this will need to be changed if we want to list all pending invitations - there will be a database table for pending invitations, and the token will contain just the ID of that table, right?